geode-wan/src/main/java/org/apache/geode/cache/client/internal/locator/wan/LocatorMembershipListenerImpl.java

```diff
@@ -74,6 +74,10 @@ public void setConfig(DistributionConfig config) {
     this.config = config;
   }
 
+  void launchLocatorsDistributorThread(Thread thread) {
```

Comment:
It seems its overdone; just to get the start in a new method for testing.
How about moving entire thread launch logic over here:
launchLocatorsDistributorThread(Thread thread) {
// Launch Locators Distributor thread.
    Runnable distributeLocatorsRunnable = new DistributeLocatorsRunnable(config.getMemberTimeout(),
        tcpClient, localLocatorId, localCopy, locator, distributedSystemId);
    ThreadFactory threadFactory =
        new LoggingThreadFactory(LOCATORS_DISTRIBUTOR_THREAD_NAME, true);
    Thread distributeLocatorsThread = threadFactory.newThread(distributeLocatorsRunnable);
    distributeLocatorsThread.start();
}

Since the test is already spying on it; you could verify if this method is called once.

Reply:
I agree in that a method to just start a thread is too much... but just verifying that the method was called isn't enough, by the time the assertions are executed the actual thread might not have been executed, the test needs an actual reference to the thread to make sure it's `TERMINATED` before executing the asserts.
I'll replace the method with another one, and use `ResultCaptor` instead of `ArgumentCaptor`, that might be better.